nemo_skills/pipeline/eval.py

```diff
@@ -202,10 +269,20 @@ def eval(
         server_type = server_type.value
     except AttributeError:
         pass
+    try:
+        extra_datasets_type = extra_datasets_type.value
+    except AttributeError:
+        pass
 
     cluster_config = get_cluster_config(cluster, config_dir)
     cluster_config = resolve_mount_paths(cluster_config, mount_paths)
 
+    if extra_datasets_type == ExtraDatasetType.cluster and cluster_config['executor'] != 'slurm':
```

Comment:
Hmm why is this ? Mounting works in docker based local env

Reply:
I think some of the tunnel logic might not work with this, so easier to just disallow for now. We can fix this later if we find it convenient to use mounted locations locally

Reply:
Sounds good